spec/features/posts_index_by_user_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe 'User Post Index', type: :feature do
```

Comment:
- [ ] As mentioned by the previous reviewer, your tests have not covered this one :point_right: `Tests that asserts you can see the first comments on a post.`